### DISCUSSION TITLE
好像没有后端代码?

### DISCUSSION THREAD

**Discussion by xiangnanscu** [ztjhz/BetterChatGPT#47] [General]
没有吧？

**Comment by ztjhz** [comment]
`ChatGPTFreeApp` is a fully frontend app that does not have a backend. However, for our free api endpoint, the backend implemented here: <https://github.com/ayaka14732/ChatGPTAPIFree>

"ChatGPTFreeApp" 是一个完全的前端应用程序，没有包含后端系统。不过，为了为我们的用户提供免费的 API 端点，我们实现了一个后端系统，可以在以下链接获得：<https://github.com/ayaka14732/ChatGPTAPIFree>
